manifests/Simon511000/Citycraftlauncher/1.9.9.yaml

```
@@ -0,0 +1,14 @@
+Id: simon511000.citycraftlauncher
```

```suggestion
Id: Simon511000.CityCraftLauncher
```

We generally use Pascal casing for the packages.